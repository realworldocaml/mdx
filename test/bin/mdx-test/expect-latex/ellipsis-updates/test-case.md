Ellipsis lines are preserved if possible even when the output changes

```sh
$ for i in `seq 1 21`; do echo $i; done
1
2
...
10
...
19
20
```

```sh
$ for i in `seq 1 19`; do echo $i; done
1
2
...
10
...
20
```
