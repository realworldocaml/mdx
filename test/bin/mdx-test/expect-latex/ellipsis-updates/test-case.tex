Ellipsis lines are preserved if possible even when the output changes

\begin{sh}
$ for i in `seq 1 21`; do echo $i; done
1
2
...
10
...
19
20
\end{sh}

\begin{sh}
$ for i in `seq 1 19`; do echo $i; done
1
2
...
10
...
20
\end{sh}
