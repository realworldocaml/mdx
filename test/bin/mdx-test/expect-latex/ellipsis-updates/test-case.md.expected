Ellipsis lines are preserved if possible even when the output changes

```sh
$ for i in `seq 1 21`; do echo $i; done
1
2
...
10
...
19
20
21
```

```sh
$ for i in `seq 1 19`; do echo $i; done
1
2
...
10
11
12
13
14
15
16
17
18
19
```
